src/pynwb/base.py

```diff
@@ -54,7 +54,7 @@ def add_container(self, **kwargs):
         '''
         container = getargs('container', kwargs)
         warn(PendingDeprecationWarning('add_container will be replaced by add_data_interface'))
```

Comment:
Change warning to `add_container will be replaced by add`

Reply:
```suggestion
        warn(PendingDeprecationWarning('add_container will be replaced by add'))
```